bool dirA
bool dirB
int16 speedA
int16 speedB